pages/news/2020-XX-XX-new-in-2-3-hotcue-colors.html

```diff
@@ -0,0 +1,59 @@
+title: New in 2.3: Color your Hotcues
+author: Holzhaus
+date: 2020-XX-XX 20:00:00
+
+{% extends "post.html" %}
+
+{% block post %}
+
+{% load markup %}
+{% filter markdown %}
+
+Hotcues are a neat thing and one of the major benefits of digital DJing.
+On analogue vinyl decks, you had had to put stickers to mark the different sections of a song and physically pick up, move and drop the needle to move to it.
+With the move to digital DJ setups, a new way to navigate tracks emerged: Hotcues now allow you to "bookmark" different spots in a track and quickly move to these positions in the blink of an eye, even while track is playing on the PA.
+
+Mixxx has supported them for more than 10 years now:
+Initial support for numbered hotcues has been added in 2009.
+
+However, it can be challenging to remember the correct number in the heat of the moment.
+Accidently pressing the wrong hotcue pad is not something can mess up your set when the track is audible.
+
+In 2015, support for hotcue labels was added to mitigate this problem.
+These can be arbitrary text that is shown on the waveforms.
```

Comment:
However, these were only shown on the scrolling waveforms, so you could not see them unless the track was close to that position. Also, it was cumbersome to edit labels because this was buried in the track Properties dialog that as only accessible from the library. Now they have been added to the overview waveforms:
[insert screenshot]
You can edit them directly from the waveform by right clicking:
[insert screenshot]
If you want long label text on hotcues that are close together, Mixxx will elide the labels so they don't overlap:
[insert screenshot]
You can still see the full label by hovering your mouse over an elided label. Mixxx will hide any other labels this would be drawn over so the text is legible:
[insert screenshot]

Reply:
Aren't you detailing this in the "Overview mouse interactions" post?

Reply:
I don't think it makes much sense to have that as a separate post. It's so closely related to this topic.

Reply:
IMHO it's not that closely related to hotcue colors and this post is already quite long. Especially when insert all the additional screenshots it is too much.
And an extra post means @ywwg get an extra opportunity to post something on social media.

Reply:
Is anyone going to care to read a post about "overview waveform interaction"? Who even knows what that means besides us? I think it makes more sense to discuss it here and in the intro/outro cue post.

Reply:
It's just a working title... something like "Make the most of your Waveform overviews" or "Manage your Cues using the waveforms". That would spark my interest since it's not obvious how.